src/components/Icon/assets/keyword.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="🔶-Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Keyword" fill="#232323">
```

Comment:
I am not sure but I think this fill needs to be removed so that that svg can be styled with different colors

Reply:
![Screenshot 2020-09-01 at 2 56 13 PM](https://user-images.githubusercontent.com/6433679/91848679-7eb61100-ec63-11ea-9412-4c882ba7c18f.png)
![Screenshot 2020-09-01 at 2 56 26 PM](https://user-images.githubusercontent.com/6433679/91848681-7f4ea780-ec63-11ea-9c57-0a2412866a1a.png)
![Screenshot 2020-09-01 at 2 56 43 PM](https://user-images.githubusercontent.com/6433679/91848682-7fe73e00-ec63-11ea-91d9-c98a120c5c52.png)